public/locales/en-US/translations.json

```diff
@@ -354,5 +354,20 @@
   "signature_reward": "Signature Reward",
   "min_account_create_amount": "Minimum Account Creation Amount",
   "other_chain_account": "Other Chain Account",
-  "xchain_claim_id": "XChain Claim ID"
+  "xchain_claim_id": "XChain Claim ID",
+  "check_nft_id": "Please check your NFT ID",
+  "get_nft_state_failed":"Unable to load NFT state",
+  "minted":"Minted",
+  "taxon_id": "Taxon ID",
+  "uri": "URI",
+  "transfer_fee": "Transfer Fee",
+  "burnable":"Burnable",
+  "only_xrp":"Only XRP",
+  "trust_line":"Trust Line",
```

Comment:
need to review this copy. does this refer to the `tfTrustLine` flag? If so, this isn't clear. something like `Creates Trust Lines for Isuser`